x-pack/plugins/ml/server/routes/saved_objects.ts

```diff
@@ -220,4 +224,50 @@ export function savedObjectsRoutes({ router, routeGuard }: RouteInitialization)
       }
     })
   );
+
+  /**
+   * @apiGroup JobSavedObjects
+   *
+   * @api {get} /api/ml/saved_objects/delete_job_check Check whether user can delete a job
+   * @apiName DeleteJobCheck
+   * @apiDescription Check the users ability to delete jobs. Returns whether they are able
+   *                 to fully delete the job and whether they are able to remove it from
+   *                 the current space.
+   *
+   * @apiSchema (body) jobIdsSchema (params) jobTypeSchema
+   *
+   */
+  router.post(
```

Comment:
Should be `DELETE` instead
```suggestion
  router.delete(
```

Reply:
it's not deleting anything. it's checking whether you can delete

Reply:
Sorry, I read it wrong! It'd be more straightforward to start the method with `Check` then :)

Reply:
yes, i can see how this could be confused.
we could change this to `check_delete_job_ability` or something like that?
@peteharverson

Reply:
Or perhaps `can_delete_job` or `check_can_delete_job` ?